core/src/components/toggle/toggle.md.vars.scss

```diff
@@ -99,3 +99,9 @@ $toggle-md-item-end-padding-bottom:               12px !default;
 
 /// @prop - Padding start of the toggle positioned on the end in an item
 $toggle-md-item-end-padding-start:                $item-md-padding-start !default;
+
+/// @prop - The text color of the on/off labels
+$toggle-md-on-off-label-color: #000 !default;
```

Comment:
These colors seem good -- we will likely need to expose a way to customize these colors in the future though

Reply:
How does the color get set for the "O" on iOS mode?

This definitely some scope creep, so feel free to ignore this for now... But what if we had a global css var for this?

For example, tab bar has a default background that can be customized with `--ion-tab-bar-background`: https://github.com/ionic-team/ionic-framework/blob/1b1b1a3800c4d044b4a3e7418f534e9271770ec6/core/src/themes/ionic.theme.default.md.scss#L17

Reason why I bring it up is iOS does change the on/off text color when adding high contrast mode.